traffic_ops/traffic_ops_golang/cachegroup/cachegroups.go

```diff
@@ -416,12 +418,22 @@ func (cg *TOCacheGroup) createCoordinate() (*int, error) {
 
 func (cg *TOCacheGroup) updateCoordinate() error {
 	if cg.Latitude != nil && cg.Longitude != nil {
-		q := `UPDATE coordinate SET name = $1, latitude = $2, longitude = $3 WHERE id = (SELECT coordinate FROM cachegroup WHERE id = $4)`
-		result, err := cg.ReqInfo.Tx.Tx.Exec(q, tc.CachegroupCoordinateNamePrefix+*cg.Name, *cg.Latitude, *cg.Longitude, *cg.ID)
+		q := `SELECT * FROM coordinate WHERE id != (SELECT coordinate FROM cachegroup WHERE id = $1) AND name =$2`
+		result, err := cg.ReqInfo.Tx.Tx.Exec(q, *cg.ID, tc.CachegroupCoordinateNamePrefix+*cg.Name)
+		if err != nil {
+			return fmt.Errorf("getting coordinate for cachegroup '%s': %s", *cg.Name, err.Error())
```

Comment:
`fmt.Errorf` formatting directive should be `%w` and consume the error, not be `%s` and consume the output of the error's `Error` method.

Reply:
Fixed.